src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -12,6 +12,10 @@ useChatCommand(builder, async (interaction: ChatInputCommandInteraction) => {
     const emoji = interaction.options.getString('emoji', true).trim();
     let emojiURL = null
 
+    if (!emoji.startsWith(`<`)) {
```

Comment:
Would probably be better to pull out `&& emoji.endsWith(">")` to here and eliminate the next if statement entirely

As a reminder, with guard clauses
```
if (!x) {
   return;
}

//we know for a fact x is true here
```

Reply:
I think I read this, didn't understand last night, and applied exactly what you wrote this morning haha! Pushed a new commit this morning.

So if I'm reading this right, since we've checked if an emoji is false, that just means emojis will always be true, meaning we don't wrap everything else in another if statement?

Reply:
(!x) where x is not a boolean will evaluate the truthiness/falsiness of a value, in our case we are making sure that `emoji` is not `null` or otherwise `undefined` (both of which would make `emoji` falsey and stop execution).

https://developer.mozilla.org/en-US/docs/Glossary/Truthy